gcloud/datastore/key.py

```diff
@@ -384,3 +369,24 @@ def parent(self):
 
     def __repr__(self):
         return '<Key%s, dataset=%s>' % (self.path, self.dataset_id)
+
+def _validate_dataset_id(dataset_id, parent):
```

Comment:
There should be two spaces between classdef and fn.def (I think lint will complain).